tests/models/test_model.py

```diff
@@ -351,7 +351,20 @@ def test_full_forward_and_backward_t5_small(batch_size: int = 2):
      pytest.param('flash', torch.float16, marks=pytest.mark.gpu),
      pytest.param('flash', torch.bfloat16, marks=pytest.mark.gpu)])
 @pytest.mark.parametrize('ffn_type', ['mptmlp', 'mptgeglu'])
-def test_determinism(attn_impl: str, precision: torch.dtype, ffn_type: str):
+@pytest.mark.parametrize('ffn_act_fn', [
+    {
+        'name': 'gelu',
+        'approximate': 'none'
+    },
+    {
+        'name': 'silu'
+    },
```

Comment:
probably just autoformatting, but always looking to keep code dense
```suggestion
    {'name': 'gelu', 'approximate': 'none'},
    {'name': 'silu'},
```

Reply:
pre-commit `yapf` undoes this...